drivers/hmis/app/models/hmis/hud/custom_data_element_definition.rb

```diff
@@ -44,6 +44,30 @@ class Hmis::Hud::CustomDataElementDefinition < Hmis::Hud::Base
 
   use_enum_with_same_key :form_role_enum_map, FIELD_TYPES.map { |f| [f, f.to_s.humanize] }.to_h
 
+  def cde_arel_field
+    cde_t = Hmis::Hud::CustomDataElement.arel_table
+    case field_type.to_sym
+    when :float
+      cde_t[:value_float]
+    when :integer
+      cde_t[:value_integer]
+    when :boolean
+      cde_t[:value_boolean]
+    when :string
+      cde_t[:value_string]
+    when :text
+      cde_t[:value_text]
+    when :date
+      cde_t[:value_date]
+    when :json
+      cde_t[:value_json]
+    when :file
+      cde_t[:value_file]
+    else
```

Comment:
Maybe we want this like this, but we could simplify a bit:

```suggestion
    case field_type.to_sym
    when :float, :integer, :boolean, :string, :text, :date, :json, :file
      cde_t["value_#{field_type}".to_sym]
    else
```

Reply:
could definitely be cleaned up. Probably a lookup hash is best, I'll add that